src/onnx/onnx.cpp

```diff
@@ -778,6 +779,50 @@ struct onnx_parser
         return add_bias(args, l1, 1);
     }
 
+    void
+    tune_padding_to_symmetric(int64_t& left, int64_t& right, const int stride, int64_t& s_start)
+    {
+        s_start = 0;
+        if(left > right)
+        {
+            right = left;
+        }
+        else if(left < right)
+        {
+            auto diff = right - left;
+            s_start   = (diff + stride - 1) / stride;
+            left      = left + s_start * stride;
+            right     = left;
+        }
+    }
+
+    template <class Op>
+    void tune_padding_size(Op& op,
```

Comment:
This should be `static`. Also, should this be templated? This looks like it only works for pooling.

Reply:
fixed